backend/alembic/versions/2023-01-03-02-57-23_21bee2994907_add_ts_vector_column_to_user_table.py

```diff
@@ -0,0 +1,32 @@
+"""add_ts_vector_column_to_user_table
+
+Revision ID: 21bee2994907
+Revises: ad172335b008
+Create Date: 2023-01-03 02:57:23.929865
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '21bee2994907'
+down_revision = 'ad172335b008'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("""
+        ALTER TABLE "user" ADD COLUMN __ts_vector__
+            tsvector GENERATED ALWAYS AS (
+                to_tsvector('english', name || ' ' || email)
```

Comment:
This is not a full-text search, see: https://akvo.semaphoreci.com/jobs/8a4a67db-93fe-49a4-b294-24bf5d945f91#L411-L423

Reply:
See the changes: https://github.com/akvo/wai-sdg-portal/compare/b57b3834ed3b5dd9e306924e4d6ef61d8e82e554..01408805df49ac69fa6e2e4ac6e4d0c6c193ccbd